docs/tfengine/schemas/resources.md

```diff
@@ -1226,6 +1226,29 @@ Normalization will make all characters alphanumeric with underscores.
 
 Type: string
 
+### storage_buckets.retention_policy
+
+Configuration of the bucket's data retention policy for how long
+objects in the bucket should be retained.
+
+Type: object
+
+### storage_buckets.retention_policy.is_locked
+
+If set to true, the bucket will be locked and permanently
+restrict edits to the bucket's retention policy.
+Caution: Locking a bucket is an irreversible action.
```

Comment:
Is this true? A bucket becomes un-deletable once locked? Not even an owner or storage admin can unlock it?

Reply:
I think so from what I played around with, but maybe you can get away with just deleting the project.

Reply:
Maybe add the link https://cloud.google.com/storage/docs/bucket-lock#overview here as well?